### ISSUE TITLE
Strange thing on rdocx object

### ISSUE THREAD

**Issue by MajoroMask** [davidgohel/officer#200]
Hi @davidgohel it's me again. I've been using `officer` and `flextable` lately and find it is convenient especially for my purpose to produce manually-editable .docx template. This time I find a problem which I'm not sure it's a bug or designed to be in this way. 

```
library(flextable)
library(officer)
library(tidyverse)

rm(list = ls())
docx_ori <- read_docx()
print(docx_ori, target = "docx_ori.docx")
docx1 <- docx_ori %>% body_add_par("a")
print(docx1, target = "docx1.docx")
docx2 <- docx_ori %>% body_add_par("a") %>% body_add_par("b")
print(docx2, target = "docx2.docx")
```

In this instance one should expect the output `docx2.docx` contains one blank line, one line of a single letter "a" and one line of a single letter "b". Yet the fact is that there are two lines of "a" instead of one. Could someone please explain this to me?

FYI: `sessionInfo()`

```
R version 3.5.2 (2018-12-20)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.3

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] forcats_0.4.0   stringr_1.4.0   dplyr_0.8.0.1   purrr_0.3.2     readr_1.3.1     tidyr_0.8.3    
 [7] tibble_2.1.1    ggplot2_3.1.0   tidyverse_1.2.1 officer_0.3.3   flextable_0.5.1

loaded via a namespace (and not attached):
 [1] tidyselect_0.2.5  xfun_0.5          haven_2.1.0       lattice_0.20-38   colorspace_1.4-1 
 [6] generics_0.0.2    htmltools_0.3.6   yaml_2.2.0        base64enc_0.1-3   rlang_0.3.2      
[11] pillar_1.3.1      withr_2.1.2       glue_1.3.1        gdtools_0.1.7     modelr_0.1.4     
[16] readxl_1.3.1      uuid_0.1-2        plyr_1.8.4        munsell_0.5.0     gtable_0.3.0     
[21] cellranger_1.1.0  rvest_0.3.2       zip_2.0.1         evaluate_0.13     knitr_1.22       
[26] broom_0.5.1       Rcpp_1.0.1        backports_1.1.3   scales_1.0.0      magick_2.0       
[31] jsonlite_1.6      hms_0.4.2         digest_0.6.18     stringi_1.4.3     grid_3.5.2       
[36] cli_1.1.0         tools_3.5.2       magrittr_1.5      lazyeval_0.2.2    crayon_1.3.4     
[41] pkgconfig_2.0.2   data.table_1.12.0 xml2_1.2.0        lubridate_1.7.4   assertthat_0.2.1 
[46] rmarkdown_1.12    httr_1.4.0        rstudioapi_0.10   R6_2.4.0          nlme_3.1-137     
[51] compiler_3.5.2  
```

**Comment by MajoroMask** [comment]
Update: 
I've noticed that in #137 @davidgohel you mentioned the R6 classes and temporary directory with xml, etc. Is this related to what I am facing here? If so, to what extent do I have to reach to avoid this? Do I have to keep no more than one `rdocx` object in the working environment? I can accept that only one `rdocx` object exists in the working environment, through it's a little hard when testing the script. 

Things is I also found that `body_replace_all_text()` function not working every third time I edit one `.docx` template which contains a `flextable` generated table in it. I can report a separate issue if needed. Looking forward to your reply, and I hope this could be of help.
